runtime/server/x86/CMakeLists.txt

```diff
@@ -122,13 +124,49 @@ else()
 endif()
 include_directories(${openfst_SOURCE_DIR}/src/include)
 
+# third_party: thrax 1.2.9 and fst archive(far) for inverse_text_normalization
+if(NOT MSVC)
+  # thrax
+  set(thrax_SOURCE_DIR ${fc_base}/thrax-src)
+  set(thrax_BINARY_DIR ${fc_base}/thrax-build)
+  ExternalProject_Add(thrax
+    URL               http://www.openfst.org/twiki/pub/GRM/ThraxDownload/thrax-1.2.9.tar.gz
+    URL_HASH          SHA256=3cfbc003609d208d50d98659166086d4a6dd341ce6697e16370f5f7ae414f2b0
+    SOURCE_DIR        ${thrax_SOURCE_DIR}
+    BINARY_DIR        ${thrax_BINARY_DIR}
+    CONFIGURE_COMMAND ${thrax_SOURCE_DIR}/configure
+                        "LDFLAGS=-L${gflags_BINARY_DIR} -L${glog_BINARY_DIR} -L${openfst_PREFIX_DIR}/lib"
+                        "CXXFLAGS=-I${gflags_BINARY_DIR}/include -I${glog_SOURCE_DIR}/src -I${glog_BINARY_DIR} -I${openfst_SOURCE_DIR}/src/include"
+                        "LIBS=-lgflags_nothreads -lglog -lpthread"
+    BUILD_COMMAND     make -j 4
+    INSTALL_COMMAND   ""
+  )
+  add_dependencies(thrax openfst)
+  target_link_libraries(utils PUBLIC ${thrax_BINARY_DIR}/src/lib/.libs/libthrax.a)
+  # download pre-built fst_archive (far)
+  execute_process(COMMAND wget https://github.com.cnpmjs.org/xingchensong/FST_ARCHIVE_FOR_ITN/releases/download/2021.07.06/itn.far -O itn.far WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
```

Comment:
The reason I chose to use `pre-built fst_archive` rather then `git clone jiayu's repo and make` is that jiayu's repo contains many other modules/grammars (such as en, ru, ...) which we don't need and fst_archive is smaller enough (about 2Mb), we can simply download it from git archives.